remoteappmanager/handlers/home_handler.py

```diff
@@ -142,10 +151,18 @@ def _actionhandler_view(self, options):
     def _actionhandler_stop(self, options):
         """Stops a running container.
         """
+        try:
+            url_id = options["url_id"][0]
+        except (KeyError, IndexError):
+            message = "Failed to retrieve valid url_id from form."
+            self.log.exception(message)
+            self.finish(message)
```

Comment:
Let is raise, line 60 will handle the error, log the exception and display a user-friendly error message in the home page